templates/ec2-asg-lch.template.yaml

```
@@ -266,4 +266,7 @@ Outputs:
     Description: Launch template Id
     Value: !Ref 'EC2LaunchTemplateResource'
     Export:
-      Name: !Sub '${AWS::StackName}-LaunchTemplateId'
\ No newline at end of file
+      Name: !Sub '${AWS::StackName}-LaunchTemplateId'
+  EC2ASGStackName:
+    Description: Stack Name of the current stack
```

Minor suggested correction to use sentence case.
```suggestion
    Description: Stack name of the current stack
```